docs/website/docs/verified-sources/workable.md

```diff
@@ -0,0 +1,232 @@
+
+The above method loads data from all the default endpoints and “candidates” endpoints to the destination.
+For the workable verified source, it's important to note that the majority of endpoints do not include the ‘*updated_at’* key. As a result, we often need to load data in '*replace*' mode, where the existing data is completely replaced. However, there is an exception for the '*candidates*' endpoints as they can be loaded incrementally, in ‘*merge’* mode.
+
+**Resource** <u>**candidate_resource:**</u>
+
+```python
+@dlt.resource(name="candidates", write_disposition="merge", primary_key="id")
+def candidates_resource(
+    updated_at: Optional[Any] = dlt.sources.incremental(
+        "updated_at", initial_value=workable.start_date_iso
+    )
+) -> Generator[TDataItems, Any, None]:
+```
+
+ 
+
+- **`updated_at`**: utilizes the *dlt.sources.incremental* method. It defaults to start_date defined in the function and if not defined defaults to Jan 1, 2000.
+
+Please note that this method is specifically defined for the “************candidates************” endpoint.
+
+### **Create Your Data Loading Pipeline**
+
+If you wish to create your own pipelines you can leverage source and resource methods in verified sources as discussed above. 
+
+To create your data pipeline using single loading and [incremental data loading](https://dlthub.com/docs/general-usage/incremental-loading) (for *********the “candidates”********* endpoint), follow these steps:
+
+1. Configure the pipeline by specifying the pipeline name, destination, and dataset. To read more about pipeline configuration, please refer to our [documentation here](https://dlthub.com/docs/general-usage/pipeline).
+
+```python
+pipeline = dlt.pipeline(
+    pipeline_name="workable_pipeline",  # Use a custom name if desired
+    destination="bigquery",  # Choose the appropriate destination (e.g., duckdb, redshift, post)
+    dataset_name="workable_dataset"  # Use a custom name if desired
+)
+```
+
+2. To load data starting from a specific date and include dependent endpoints, you can utilize the 'start_date' parameter and set it to the desired date. For example, to load data from January 1, 2022, and include all corresponding details for dependent endpoints:
+
+```python
+load_data = workable_source(start_date=datetime(2022, 1, 1), load_details=True)
+```
+
+> Note: The **`load_details`** parameter controls whether the [**dependent endpoints]**(workable.md#dependent-endpoints) are loaded. If you want to load the dependent endpoints, set this parameter to **`True`** in your pipeline run. If not, **`False`**.
+> 
+
+3. Use the method **`pipeline.run()`** to execute the pipeline.
+
+```python
+load_info = pipeline.run(load_data)
+    # print the information on data that was loaded
+    print(load_info)
+```
+
+4. To use the method `pipeline.run()` to load custom endpoints “candidates” and “*members*”, the above script may be modified as:
```

Comment:
sorry I meant there was wrong order of * and `

it was \` * * pipeline.run() * * \`

should be * * \` pipeline.run() \` * *

Reply:
done